Octokit.Tests.Integration/fixtures/RepositoriesHooksCollection.cs

```diff
@@ -0,0 +1,9 @@
+﻿using Xunit;
+
+namespace Octokit.Tests.Integration.fixtures
+{
+    [CollectionDefinition("Repositories Hooks Collection")]
+    public class RepositoriesHooksCollection : ICollectionFixture<RepositoriesHooksFixture>
```

Comment:
cc @haacked as this might be a better way for us to simplify the setup and teardown of repositories in tests that we were throwing around in #655